Modules/_elementtree.c

```diff
@@ -3338,7 +3339,6 @@ _elementtree_XMLParser___init___impl(XMLParserObject *self, PyObject *target,
         if (!target) {
             Py_CLEAR(self->entity);
             Py_CLEAR(self->names);
-            EXPAT(ParserFree)(self->parser);
```

Comment:
Why this is removed?

Reply:
`self->parser` would be freed again during deallocation, which would cause a crash.